### ISSUE TITLE
Number of hits before searching..Feature Will be awesome..

### ISSUE THREAD

**Issue by jkathir** [giuspen/cherrytree#277]
Google like search engine with number of hits displayed before searching..It would be great...

Even google doesn't have the number of hits....

But number of hits are done and possible

ex: http://table.branham.org/#/main

Type any word and wait for 2 second it will show the number of hits possible for the given word or sentence.

**Comment by philippHRO** [comment]
Neat. 
This could be very useful for people with huge Cherry Tree Documents (like me ;)
![grafik](https://user-images.githubusercontent.com/23098362/29319881-d2ac726e-81d5-11e7-99e2-d9afb3dbd31d.png)

**Comment by jkathir** [comment]
Yes, philippHRO..it would be nice and great...Hope it would be taken into consideration and implement it soon..

**Comment by jkathir** [comment]
Did you get a chance to work on this ???

**Comment by jkathir** [comment]
Did you get a chance to work on this ???

**Comment by EllieBorden** [comment]
@jkathir Cherrytree's developmental focus is on porting to C++/GTKmm at this time.
